src/Functions/generateULID.cpp

```diff
@@ -74,18 +74,44 @@ class FunctionGenerateULID : public IFunction
 
 REGISTER_FUNCTION(GenerateULID)
 {
-    factory.registerFunction<FunctionGenerateULID>(FunctionDocumentation
+    /// generateULID documentation
+    FunctionDocumentation::Description description_generateULID = R"(
+Generates a [Universally Unique Lexicographically Sortable Identifier (ULID)](https://github.com/ulid/spec).
+    )";
+    FunctionDocumentation::Syntax syntax_generateULID = "generateULID([x])";
+    FunctionDocumentation::Arguments arguments_generateULID = {
+        {"x", "Optional. An expression resulting in any of the supported data types. The resulting value is discarded, but the expression itself if used for bypassing common subexpression elimination if the function is called multiple times in one query.", {"Any"}}
```

Comment:
Same comment as in generateUUIDv7.cpp (--> link to common subexpression removal).